SimulationAgent/Runtime/Config.cs

```diff
@@ -46,13 +53,23 @@ public Config(IConfigData configData)
                                     "value in the 'appsettings.ini' configuration file.");
             }
 
+            var limitsConf = new RateLimitingConfiguration
+            {
+                ConnectionsPerSecond = configData.GetDouble(CONNECTIONS_FREQUENCY_LIMIT_KEY),
+                RegistryOperationsPerMinute = configData.GetDouble(REGISTRYOPS_FREQUENCY_LIMIT_KEY),
+                MessagesPerDay = configData.GetDouble(MESSAGES_FREQUENCY_LIMIT_KEY),
+                TwinReadsPerSecond = configData.GetDouble(TWIN_READS_FREQUENCY_LIMIT_KEY),
+                TwinWritesPerSecond = configData.GetDouble(TWIN_WRITES_FREQUENCY_LIMIT_KEY)
```

Comment:
Where is the throttle config for Device-to-cloud sends?  For S1 == Higher of 100/sec or 12/sec/unit 

https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-quotas-throttling

Reply:
we don't use c2d messages in the simulation, right?